src/main/java/CleanCycle/Analytics/Main.java

```diff
@@ -466,12 +466,23 @@ public void run() {
             }
 
             synchronized (points) {
-                try {
-                    loadPointsFromDatabase(points);
-                }
-                catch (IOException | ParseException e) {
-                    System.out.println("There was an error getting points from the database.");
-                    e.printStackTrace();
+                 exponentialBackoff = 1; // wait longer durations of time every time unsuccessful.
```

Comment:
@Metasynic I've added exponential backoff to your subsequent endpoint connection attempts, perhaps when you refactor the code you could incorporate this into the loadPointfromDB() method?

Reply:
Yes, that's a good suggestion.